post-scan-actions/aws-python-promote-or-quarantine/README.md

```diff
@@ -103,26 +103,46 @@ After a scan occurs, this example Lambda function places clean files in one buck
         - Make sure that two policies are listed.
         - Click **Create role**.
     </details>
-    
+
    <details>
    <summary>Using the AWS CLI</summary>
 
-    1. Enter the following AWS CLI command to create the role:
+    1. Paste the [JSON code below](#JSON_trust-doc) into a file called `trust.json` (or another name).
+    2. Enter the following AWS CLI command to create the role:
 
         `aws iam create-role --role-name <YOUR_FSS_LAMBDA_ROLE>`
```

Comment:
Looks like you missed the step to use the `trust.json`. How about adding `--assume-role-policy-document` to `create-role` command?

Reply:
@carpusherw ironic :) anyway good catch, I was working off a copy paste draft and forgot it but just pushed it. Thank you.